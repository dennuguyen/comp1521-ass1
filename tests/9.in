60
33
1