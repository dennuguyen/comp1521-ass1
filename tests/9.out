Enter world size: Enter rule: Enter how many generations: 
0	..............................#.............................
1	#############################...############################
